portals/devportal/src/main/webapp/site/public/theme/userTheme.json

```diff
@@ -1,7 +1,55 @@
 {
     "custom": {
         "landingPage": {
-            "active": false
+            "active": false,
```

Comment:
I think we can keep the userTheme.json file minimal for now. Users who wish to customize the theming can refer to the documentation for detailed instructions. Could we please revert the recent changes to this file so it remains as-is?